Kickstarter-iOS/Views/Controllers/LandingPageViewController.swift

```diff
@@ -0,0 +1,228 @@
+    let spacer2 = UIView()
+    let spacer3 = UIView()
+
+    _ = ([
+      spacer1,
+      self.labelsStackView,
+      self.cardsStackView,
+      spacer2,
+      spacer3,
+      self.ctaButton
+    ], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = (self.rootStackView, self.view)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToMarginsInParent()
+  }
+
+  private func setupConstraints() {
+    NSLayoutConstraint.activate([
+      self.scrollView.heightAnchor.constraint(greaterThanOrEqualToConstant: Layout.Card.height),
+      self.cardViewsStackView.heightAnchor.constraint(equalTo: self.scrollView.heightAnchor)
+    ])
+  }
+
+  @objc private func ctaBUttonTapped() {
+    self.dismiss(animated: true)
+  }
+
+  private func configureCards(with _: [UIView]) {}
+}
+
+// Styles
+private let backgroundImageViewStyle: ImageViewStyle = { imageView in
+  imageView
+    |> \.contentMode .~ .scaleToFill
+    |> \.image .~ image(named: "landing-page-background")
+}
+
+private let cardsStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> verticalStackViewStyle
+}
+
+private let ctaButtonStyle: ButtonStyle = { button in
+  button
+    |> greenButtonStyle
+    |> UIButton.lens.title(for: .normal) %~ { _ in
+      localizedString(key: "Get_started", defaultValue: "Get started")
+    }
+    |> \.translatesAutoresizingMaskIntoConstraints .~ false
+}
+
+private let descriptionLabelStyle: LabelStyle = { label in
+  label
+    |> \.numberOfLines .~ 0
+    |> \.textColor .~ UIColor.ksr_text_dark_grey_500
+    |> \.font .~ UIFont.ksr_callout()
+    |> \.textAlignment .~ .center
+    |> \.text %~ { _ in
+      Strings.Pledge_to_projects_and_view_all_your_saved_and_backed_projects_in_one_place()
+    }
+}
+
+private let labelsStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> verticalStackViewStyle
+    |> \.spacing .~ Styles.grid(3)
+}
+
+private let logoImageViewStyle: ImageViewStyle = { imageView in
+  imageView
+    |> \.image .~ image(named: "kickstarter-logo")?.withRenderingMode(.alwaysTemplate)
+    |> \.tintColor .~ .ksr_green_400
+    |> \.contentMode .~ .scaleAspectFit
+    |> \.translatesAutoresizingMaskIntoConstraints .~ false
+}
+
+private let rootStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> verticalStackViewStyle
+    |> \.spacing .~ Styles.grid(5)
+    |> \.translatesAutoresizingMaskIntoConstraints .~ false
+    |> \.isLayoutMarginsRelativeArrangement .~ true
+    |> \.layoutMargins .~ .init(top: Styles.grid(15), left: 0, bottom: Styles.grid(3), right: 0)
+    |> \.distribution .~ .equalSpacing
+}
+
+private let scrollViewStyle: ScrollStyle = { scrollView in
+  scrollView
+    |> \.bounces .~ false
+    |> \.isPagingEnabled .~ true
+}
+
+private let titleLabelStyle: LabelStyle = { label in
+  label
+    |> \.text %~ { _ in Strings.Bring_creative_projects_to_life() }
+    |> \.font .~ UIFont.ksr_title3().bolded
+    |> \.textAlignment .~ .center
```

Comment:
Do we want to this to wrap with larger font sizes? Right now it's truncating.

![Simulator Screen Shot - iPhone X - 2020-02-26 at 13 14 47](https://user-images.githubusercontent.com/3156796/75374289-fd3b8b80-5899-11ea-91c4-1127eb60c905.png)